src/WorkflowCore/Models/WorkflowOptions.cs

```diff
@@ -67,6 +70,16 @@ public void UseErrorRetryInterval(TimeSpan interval)
         {
             ErrorRetryInterval = interval;
         }
+
+        public void UseMaxConcurrentItems(int? maxConcurrentItems)
```

Comment:
Perhaps we should make this explicitly the max number of concurrent workflows rather than items?

Reply:
Let me make that change you've suggested.  Most consumers of your API may not ever dig into the internals, and thus not know what maxConcurrentItems really means, but framing it as "max number of concurrent workflows" is more intuitive.

Reply:
I think instead of modifying EventConsumer directly, I'll move the change to WorkflowConsumer...that way the other child consumers can keep their default behavior the way you intended.